sml_small/editing/thousand_pounds/README.md

```diff
@@ -68,24 +71,23 @@ output = thousand_pounds(
 ```python
 Thousands_output(
     principal_identifier='12340000001-201409-q100',
-    principal_original_value=50000000,
-    principal_adjusted_value=50000.0,
+    principal_adjusted_value="50000.0",
     target_variables=[
-        Target_variable(identifier='101', original_value=500, adjusted_value=0.5),
-        Target_variable(identifier='102', original_value=1000, adjusted_value=1.0),
-        Target_variable(identifier='103', original_value=1500, adjusted_value=1.5),
+        Target_variable(identifier='101', original_value="500", adjusted_value="0.5"),
+        Target_variable(identifier='102', original_value="1000", adjusted_value="1.0"),
+        Target_variable(identifier='103', original_value="1500", adjusted_value="1.5"),
         Target_variable(identifier='104', original_value=None, adjusted_value=None)],
-    tpc_ratio=833.3333333333334,
+    tpc_ratio="833.3",
     tpc_marker='C',
     error_description=''
 )
 ```
 
 Alternatively presented: (mixing input and output data for comparisons)
 
-|principal_identifier|principal_variable|predictive|aux|threshold_upper|threshold_lower|TPC_marker|ratio|principal_adjusted_value|target_variable|target_original_value|target_adjusted_value
-|---|---|---|---|---|---|---|---|---|---|---|---|
-12340000001-201409-q100|50000000|60000|15000|1350|350|C|1000.0|50000.0|q101|500|0.5
-12340000001-201409-q100|-|-|-|-|-|-|-|-|q102|1000|1
-12340000001-201409-q100|-|-|-|-|-|-|-|-|q103|1500|1.5
-12340000001-201409-q100|-|-|-|-|-|-|-|-|q104||
+| unique_identifier       |principal_variable|predictive|aux|threshold_upper|threshold_lower|TPC_marker|ratio|principal_adjusted_value|target_variable|target_original_value|target_adjusted_value
+|-------------------------|---|---|---|---|---|---|---|---|---|---|---|
+ 12340000001-201409-q100 |50000000|60000|15000|1350|350|C|1000.0|50000.0|q101|500|0.5
+ 12340000001-201409-q100 |-|-|-|-|-|-|-|-|q102|1000|1
```

Comment:
should there be tpc values here?

Reply:
tpc marker and ratio is in there, the table view makes a bit more sense all rows are a single test case, its just how the different target variables have been represented

Reply:
table representation is a bit weird here, the entire table is a single test case, with each row representing the principal variable and each target variable, so tpc marker and ratio are there for the first row,

amended ratio name (should be tpc_ratio), also removed target_original_value as no longer in output

Reply:
Ok happy with this will do a read through and approve it from there :)